filebeat/filebeat.reference.yml

```diff
@@ -723,6 +726,9 @@ filebeat.inputs:
   # Combine partial lines flagged by `json-file` format
   #combine_partials: true
 
+  # Keep the original unparsed line under log.original field.
+  #keep_original_message: false
```

Comment:
What if we keep the same name as above? One settings and clearly make it link related to JSON?

I mean `json.keep_original_message` instead of `keep_original_message`

Reply:
I suspect we would like to allow user to override the 'original' field.

Reply:
I don't think it is relevant information regarding the Docker input. I think if someone uses this input rather than `log` input with appropiate `json.*` configuration, he/she does not care whether it is JSON, XML or CSV. He/she only wants Docker logs shipped to the configured output. Also, one of the key advantages of using `docker` input instead of `log` is that we can "cover" such implementation details.

Reply:
Well if the intent is to have the same config `keep_original_message` drive this feature of copying the original event to `log.original` -- regardless of format -- then I agree it should not be nested under `json.`.

Reply:
Pretty good your points, OK let's hide the fact that we use json.

It was more in the context of adding the target field and trying to keep the same naming.